graph/typeDefs.graphql

```diff
@@ -838,6 +847,10 @@ type RootQuery {
   # expensive as it is not batched. Requires the `ADMIN` role.
   commentCount(query: CommentCountQuery!): Int
 
+  # Return the count of users satisfied by the query. Note that this edge is
+  # expensive as it is not batched. Requires the `ADMIN` role.
```

Comment:
Instead of `Requires the ADMIN role` we decided to only write `This field is restricted` until we have a more maintainable way of automatically generating that information.